### PR TITLE
Fix logo size and alignment and allow text as logo

### PR DESCRIPTION
**Pull request by amf9t2**
logo_flex_width

### CODE REVIEW

File: packages/terra-consumer-nav/src/components/nav-logo/NavLogo.scss [hunk 1]
```diff
@@ -6,17 +6,24 @@
...
     margin: auto;
     max-height: 120px;
     max-width: 100%;
   }
+
+  .logo-text {
+    color: var(--terra-consumer-logo-text-color, #3d3d3d);
+    font-size: 30px;
+    font-weight: normal;
+    text-shadow: 0.25px 0.25px;
```
Comment (KevinJJackson) [inline packages/terra-consumer-nav/src/components/nav-logo/NavLogo.scss]: wouldn't setting these on the .img class be applied to the alternate text anyway? why make the second class?

File: packages/terra-consumer-nav/src/components/nav-logo/NavLogo.jsx [hunk 2]
```diff
@@ -32,14 +32,19 @@ const NavLogo = ({
   isCard,
   ...customProps
 }) => {
-  const image = <img className={cx('img')} src={url} alt={altText} />;
+  let logoContent;
+  if (url) {
+    logoContent = <img className={cx('img')} src={url} alt={altText} />;
+  } else {
+    logoContent = (<p className={cx('logo-text')}>{altText}</p>);
```
Comment (KevinJJackson) [inline packages/terra-consumer-nav/src/components/nav-logo/NavLogo.jsx]: the purpose of the altText is that it is displayed when no url is provided or the image fails to load, this shouldnt be needed.

Comment (amf9t2) [reply packages/terra-consumer-nav/src/components/nav-logo/NavLogo.jsx]: make sense I added the required styles to img.

File: packages/terra-consumer-nav/src/components/nav-logo/NavLogo.scss (hunk 1 above)
Comment (amf9t2) [reply packages/terra-consumer-nav/src/components/nav-logo/NavLogo.scss]: Anna said we don't want text shadow for now so removed it.

**Comment by zhongyn** [comment]
can you add a test like the simple nav example using text logo?

**Comment by amf9t2** [comment]
@zhongyn [added](https://github.com/cerner/terra-consumer/pull/50/commits/83aa0d066d602c58cc6f7d59ba3eae63eaac33ef)

**Comment by amf9t2** [comment]
logo_text

File: packages/terra-consumer-nav/src/components/nav-logo/NavLogo.scss [hunk 3]
```diff
@@ -6,15 +6,17 @@
...
+    text-align: center;
     @media screen and (max-width: $terra-medium-breakpoint) {
       margin: 40px 25px;
     }
   }
 
   .img {
-    color: #3d3d3d;
-    display: block;
+    color: var(--terra-consumer-logo-text-color, #3d3d3d);
```
Comment (alex-bezek) [inline packages/terra-consumer-nav/src/components/nav-logo/NavLogo.scss]: why does an image need a text color??

**Comment by alex-bezek** [comment]
@mhemesath when would a client want a plain text logo rather than a png?

File: packages/terra-consumer-layout/src/Layout.scss [hunk 4]
```diff
@@ -11,6 +11,7 @@
   --terra-consumer-profile-link-width: var(--terra-consumer-nav-width);
   --terra-consumer-profile-link-background: var(--terra-consumer-body-background);
   --terra-consumer-profile-link-color: var(--terra-consumer-nav-item-color);
+  --terra-consumer-logo-text-color: var(--terra-consumer-nav-item-color);
```
Comment (mhemesath) [inline packages/terra-consumer-layout/src/Layout.scss]: We don't need a logo text color prop... it should just be the text color of the nav

Comment (mhemesath) [reply packages/terra-consumer-layout/src/Layout.scss]: We can merge this for now and defer this when we do custom prop cleanup...

**Pull request merged**